cfpb/cfgov-refresh
# Update BAH dependencies

Updates outdated OAH/BAH app dependencies.

## Changes

- Updates `axios`, `amortize`, `date-format`, `es6-promise`, `fastdom`, `format-usd`, `handlebars`, and `highcharts`.
- Updates format-usd syntax to fit with newest API changes.

## Testing

1. Run `./frontend.sh`
2. Run `gulp clean && gulp build`
3. Compare functionality of http://localhost:8000/owning-a-home/explore-rates/ to https://www.consumerfinance.gov/owning-a-home/explore-rates/
In particular, download charts should function correctly. $ values should display correctly. When entering a high loan amount (e.g. $600,000), the county drop-down should show and the county warnings text (coming from handlebars templates) should show. Tool should also still work in IE11.
